### PR TITLE
Fix TachiyomiAZ.md

### PR DESCRIPTION
**Pull request by az4521**

### CODE REVIEW

**Comment by AbdullahM0hamed** [comment]
Speech 100😂😂😂😂😂😂

File: src/forks/TachiyomiAZ.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ sidebar: false
 
 <ForkButtons forkName="TachiyomiAZ" downloadForkLink="https://api.github.com/repos/az4521/TachiyomiAZ/releases/latest" downloadForkStyle="background-color:#FFCC4D;color:#000000;" githubForkLink="window.open('https://github.com/az4521/TachiyomiAZ')"/>
 
+<marquee>The BEST fork</marquee>
```
Comment (Soitora) [inline src/forks/TachiyomiAZ.md:21]:
```suggestion
<marquee><i>"The BEST fork" - az4521</i></marquee>
```

**Comment by joseph619** [comment]
> Don't really feel like this belongs here

How so? AZ can claim his fork is the best on his own page

**Comment by AbdullahM0hamed** [comment]
He can claim his fork is the best sure - but that doesn't mean the site should endorse his claim.

User's should have an unbiased presentation of the forks so they can decide which they prefer, instead of the site giving them a nudge in the direction of a specific fork.

File: src/forks/TachiyomiAZ.md (hunk 1 above)
Comment (AbdullahM0hamed) [reply src/forks/TachiyomiAZ.md:21]: I like this addition.

**Comment by GHOSTBEAR** [comment]
> > Don't really feel like this belongs here
> 
> How so? AZ can claim his fork is the best on his own page

Sure he may claim that his fork is the best, but I feel like this is bloat and unnecessary on the site. But if @Soitora want to approve this then fine

**Comment by Jays2Kings** [comment]
There's no bias here, all I see is a user stating a fact and placing it where it needs to be

**Comment by Soitora** [comment]
As long as he makes it clear that it is he *the creator* saying it, I don't believe it's too out of place. I want the Forks page to be customizable to their liking (with limits). I'm gray on this but other people seem fine with it so I don't mind.

**Comment by AbdullahM0hamed** [comment]
I see the author of *the actual best fork* is also trying to misdirect users, in order to get less issues.

Well played, sir, well played.

**Comment by GHOSTBEAR** [comment]
> As long as he makes it clear that it is he _the creator_ saying it, I don't believe it's too out of place. I want the Forks page to be customizable to their liking (with limits). I'm gray on this but other people seem fine with it so I don't mind.

Sure if it's saying that it is the creator is saying this then I can agree on it. But can we keep it to normal text and not have a marquee element

**Comment by AbdullahM0hamed** [comment]
>>> normal text
Normalcy is not in Az's dictionary.

**Comment by AbdullahM0hamed** [comment]
> normal text
Normalcy is foreign to az.

**Comment by GHOSTBEAR** [comment]
Well, the marquee element is obsolete in HTML5. So if he would like to write it with some JavaScript and CSS then he can keep it

**Comment by Soitora** [comment]
> Well, the marquee element is obsolete in HTML5. So if he would like to write it with some JavaScript and CSS then he can keep it

Agreed

**Comment by az4521** [comment]
<marquee> is great, and even if its obsolete it's still supported in every major browser

i *will* implement rainbow text that jumps around if you make me use CSS3 and JS

**Comment by az4521** [comment]
i take it there are no more complaints and that it's ready to be merged?

**Comment by Soitora** [comment]
Up to @GHOSTBEAR, for me it's fine. It's "not a part" of regular Tachiyomi anyway

File: src/forks/TachiyomiAZ.md [hunk 2]
```diff
@@ -13,10 +13,76 @@ meta:
     content: "#FFCC4D"
 sidebar: false
 ---
+<style lang="css">
+@font-face {font-family: "Comic Sans MS"; src: url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.eot"); src: url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.eot?#iefix") format("embedded-opentype"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.woff2") format("woff2"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.woff") format("woff"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.ttf") format("truetype"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.svg#Comic Sans MS") format("svg"); }
+
+.seizureTextAJDAASDASDASDASd {
```
Comment (GHOSTBEAR) [inline src/forks/TachiyomiAZ.md:19]: Rename this to `.seizureText`

File: src/forks/TachiyomiAZ.md [hunk 3]
```diff
@@ -13,10 +13,76 @@ meta:
...
+  animation: RAINBOOOWWWWWWWWW .5s linear infinite, VengaboysUPANDdownandUPANDdown 2s infinite;
+}
+.leanAndDab {
+  padding: 2em;
+  width: fit-content;
+  animation: GOLEFTGORIGHTGOPICKUPTHESTEPGOLEFTGORIGHTGOLEFT 1s ease-out infinite;
+}
+.bigBoyText {
```
Comment (GHOSTBEAR) [inline src/forks/TachiyomiAZ.md:27]: Rename this to `.bigText`

File: src/forks/TachiyomiAZ.md [hunk 4]
```diff
@@ -13,10 +13,76 @@ meta:
...
+  50%  {transform: rotate(0deg)}
+  75%  {transform: rotate(15deg)}
+  100% {transform: rotate(0deg)}
+}
+@keyframes VengaboysUPANDdownandUPANDdown {
+  0% {transform: scaleX(.5) scaleY(.5)}
+  50% {transform: scaleX(1) scaleY(1)}
+  100% {transform: scaleX(.5) scaleY(.5)}
+}
+@keyframes RAINBOOOWWWWWWWWW {
```
Comment (GHOSTBEAR) [inline src/forks/TachiyomiAZ.md:61]: Rename this to `rainbowText`

File: src/forks/TachiyomiAZ.md [hunk 5]
```diff
@@ -13,10 +13,76 @@ meta:
...
+  100% {transform: translate(-100%, 0);}
+}
+@keyframes GOLEFTGORIGHTGOPICKUPTHESTEPGOLEFTGORIGHTGOLEFT {
+  0%   {transform: rotate(0deg)}
+  25%  {transform: rotate(-15deg)}
+  50%  {transform: rotate(0deg)}
+  75%  {transform: rotate(15deg)}
+  100% {transform: rotate(0deg)}
+}
+@keyframes VengaboysUPANDdownandUPANDdown {
```
Comment (GHOSTBEAR) [inline src/forks/TachiyomiAZ.md:56]: Rename this to `.upAndDownAnimation`

File: src/forks/TachiyomiAZ.md [hunk 6]
```diff
@@ -13,10 +13,76 @@ meta:
...
+}
+.containTheBuldgeOwO {
+	width:100%;
+	overflow:hidden;
+}
+@keyframes YEEEEEEEEEEEEEEEEET {
+  0% {transform: translate(0, 0);}
+  100% {transform: translate(-100%, 0);}
+}
+@keyframes GOLEFTGORIGHTGOPICKUPTHESTEPGOLEFTGORIGHTGOLEFT {
```
Comment (GHOSTBEAR) [inline src/forks/TachiyomiAZ.md:49]: Rename this to `textWiggleAnimation`

File: src/forks/TachiyomiAZ.md [hunk 7]
```diff
@@ -13,10 +13,76 @@ meta:
...
+  overflow: hidden;
+  white-space: nowrap;
+  animation: YEEEEEEEEEEEEEEEEET 10s linear infinite;
+  padding-left: 100%;
+}
+.containTheBuldgeOwO {
+	width:100%;
+	overflow:hidden;
+}
+@keyframes YEEEEEEEEEEEEEEEEET {
```
Comment (GHOSTBEAR) [inline src/forks/TachiyomiAZ.md:45]: Rename this to `textLoopAnimation`

File: src/forks/TachiyomiAZ.md [hunk 8]
```diff
@@ -13,10 +13,76 @@ meta:
...
+  text-shadow: -1px -1px 0 #000, 1px -1px 0 #000, -1px 1px 0 #000, 1px 1px 0 #000;
+}
+.hahaSlideyBoi {
+  display: inline-block;
+  overflow: hidden;
+  white-space: nowrap;
+  animation: YEEEEEEEEEEEEEEEEET 10s linear infinite;
+  padding-left: 100%;
+}
+.containTheBuldgeOwO {
```
Comment (GHOSTBEAR) [inline src/forks/TachiyomiAZ.md:41]: Rename this to `.customMarquee`

File: src/forks/TachiyomiAZ.md [hunk 9]
```diff
@@ -13,10 +13,76 @@ meta:
...
+  animation: GOLEFTGORIGHTGOPICKUPTHESTEPGOLEFTGORIGHTGOLEFT 1s ease-out infinite;
+}
+.bigBoyText {
+  font-family: "Comic Sans MS", "Comic Sans", cursive;
+  font-size: 2em;
+  display: inline-block;
+  -webkit-text-stroke: 1px black;
+  text-shadow: -1px -1px 0 #000, 1px -1px 0 #000, -1px 1px 0 #000, 1px 1px 0 #000;
+}
+.hahaSlideyBoi {
```
Comment (GHOSTBEAR) [inline src/forks/TachiyomiAZ.md:34]: Rename this to `.textLoop`

File: src/forks/TachiyomiAZ.md [hunk 10]
```diff
@@ -13,10 +13,76 @@ meta:
     content: "#FFCC4D"
 sidebar: false
 ---
+<style lang="css">
+@font-face {font-family: "Comic Sans MS"; src: url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.eot"); src: url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.eot?#iefix") format("embedded-opentype"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.woff2") format("woff2"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.woff") format("woff"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.ttf") format("truetype"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.svg#Comic Sans MS") format("svg"); }
+
+.seizureTextAJDAASDASDASDASd {
+  animation: RAINBOOOWWWWWWWWW .5s linear infinite, VengaboysUPANDdownandUPANDdown 2s infinite;
+}
+.leanAndDab {
```
Comment (GHOSTBEAR) [inline src/forks/TachiyomiAZ.md:22]: Rename this to `.textWiggle`

File: src/forks/TachiyomiAZ.md [hunk 11]
```diff
@@ -13,10 +13,76 @@ meta:
     content: "#FFCC4D"
 sidebar: false
 ---
+<style lang="css">
+@font-face {font-family: "Comic Sans MS"; src: url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.eot"); src: url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.eot?#iefix") format("embedded-opentype"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.woff2") format("woff2"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.woff") format("woff"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.ttf") format("truetype"), url("//db.onlinewebfonts.com/t/7cc6719bd5f0310be3150ba33418e72e.svg#Comic Sans MS") format("svg"); }
```
Comment (GHOSTBEAR) [inline src/forks/TachiyomiAZ.md:17]: Would prefer if you used Google Fonts [Comic Neue](https://fonts.google.com/specimen/Comic+Neue?query=comic)

File: src/forks/TachiyomiAZ.md [hunk 12]
```diff
@@ -13,10 +13,76 @@ meta:
...
 # <img class="headerLogo" :src="$withBase('/forks/TachiyomiAZ/img/logo.png')"> TachiyomiAZ
 
 <ForkButtons forkName="TachiyomiAZ" downloadForkLink="https://api.github.com/repos/az4521/TachiyomiAZ/releases/latest" downloadForkStyle="background-color:#FFCC4D;color:#000000;" githubForkLink="window.open('https://github.com/az4521/TachiyomiAZ')"/>
+<div class="containTheBuldgeOwO">
+  <div class="hahaSlideyBoi">
+    <div class="leanAndDab">
+      <span class="seizureTextAJDAASDASDASDASd bigBoyText"><i>The BEST fork</i> --az4521</span>
+    </div>
+  </div>
+</div>
```
Comment (GHOSTBEAR) [inline src/forks/TachiyomiAZ.md:85]: Don't forget to rename the classes here to

**Pull request merged**